Sim/SimPapasDetector/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+################################################################################
```

Comment:
I don't know the software architecture of FCCSW very well, but why is SimPapasDetector not in Sim/SimPapas?

Reply:
I think it would be good to discuss with @benedikt the best place to put the papas files within the FCC structures.

Reply:
I have moved as you have suggested